packages/litqa/README.md

```diff
@@ -0,0 +1,70 @@
+# aviary.litqa
+
+LitQA2 environment implemented with aviary,
+allowing agents to perform question answering on the LitQA dataset.
+
+LitQA is a is a dataset composed from 50 multiple-choice questions from recent literature.
```

Comment:
Just say something like: `LitQA (now legacy) is a`

Also, duplicate `is a is a`